<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lab3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>19</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;华文行楷&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>              自动报警系统</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{
	border-image: url(:/images/beijing13.jpg);
}；

#stackedWidget *{
	border-image: url( );
}</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 18pt &quot;华文行楷&quot;;</string>
     </property>
     <property name="text">
      <string>AutoMatic Alarm System</string>
     </property>
    </widget>
    <widget class="QToolButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;</string>
     </property>
     <property name="text">
      <string>链接</string>
     </property>
    </widget>
    <widget class="QToolButton" name="disconnect_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;</string>
     </property>
     <property name="text">
      <string>断开链接</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo1_lab">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>330</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(223, 223, 167);
font: 75 italic 20pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_lab">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(220, 220, 165);
font: 75 italic 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>lab3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
